app/views/volunteer_mailer/probation_period_reminder.html.slim

```diff
@@ -0,0 +1,2 @@
+h1= @subject
```

Comment:
no, that doesn't make sense at all, sorry. Because I'd need to partialize the view for web and it would make the thing totally unreadable. because the mailing only wants two elements `h1` and `div`.

Reply:
ah, now I see. The file `app/views/reminder_mailings/probation_period_reminder.html.slim` is useless where it is. I'll move it. Thanks!